app/javascript/src/NewService/components/ServiceNewForm.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import React from 'react'
+// TODO: Get authenticityToken from rails
+const authenticityToken = ''
```

Comment:
commented above regarding getting this. You could extract that function to `utils`